CubeScramble.py

```diff
@@ -4,20 +4,22 @@
 # Example: R, Ri or R2
 
 
-
-
-import random
-
+from random import choice
 
 # all possible moves on a Rubik's Cube
 
-list = ['U', 'Ui', 'U2', 'F', 'Fi', 'F2', 'R', 'Ri', 'R2', 'B', 'Bi', 'B2', \
+list = ['U', 'Ui', 'U2', 'F', 'Fi', 'F2', 'R', 'Ri', 'R2', 'B', 'Bi', 'B2',
```

Comment:
I didn't update this because I wanted you to focus more on the `\`, but constants are all upper case. Also, avoid anything that is already a keyword. Also, "list" doesn't tell you shit about what it is. be descriptive. You'll be glad you were when this file is a thousand lines. Maybe `CUBE_TURNS`?

Reply:
Good point.